### DISCUSSION TITLE
how to save best child net and load for inference

### DISCUSSION THREAD

**Discussion by iksooman** [microsoft/nni#4215] [Q&A] [answered]
I tested enas code of nas example. 
After fitting trainer, I founded there's no best child net saving code.
Could you tell me how to save the best child net and load for inference?

**Comment by ultmaster** [comment, marked as answer]
We have a export in the trainer, which will export the searched architecture and saved it in the json.

**Comment by iksooman** [comment]
@ultmaster But the saved json contains only selected choices by controller. right?
Then is it possible to save trained child network using the selected choices?

**Comment by ultmaster** [comment]
This is currently not supported by framework and I'm afraid that you have to do this manually. I'm also curious why you need this feature because as far as I know the child network weights produced by ENAS doesn't actually make any sense.

**Comment by iksooman** [comment]
@ultmaster Thank you for your reply.
I've thought that what we want from NAS including ENAS is the best child network.
(I'm not an expert about NAS. So, I might have wrong knowledge of this.)
